sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -146,18 +140,15 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    */
   @scala.annotation.varargs
   def load(paths: String*): DataFrame = {
-    if (paths.isEmpty) {
```

Comment:
In my PR, will add the test cases to verify all the possible inputs after this code changes. Thanks!